packages/components/src/patterns/quote/quote.stories.ts

```diff
@@ -0,0 +1,175 @@
+import '../../solid-components';
+import { html } from 'lit-html';
+
+export default {
+  title: 'Pattern/quote',
+  component: 'Quote',
+  parameters: {
+    chromatic: { disableSnapshot: true },
+    design: {
+      type: 'figma',
+      url: 'https://www.figma.com/file/hER2N8wZXhiTrdlDXyrdUt/Quote?type=design&node-id=1001-4293&mode=design&t=Xywhix1rQMoatokH-0'
+    }
+  }
+};
+
+const img = (hidden = false) => html`
+  <img
+    class="w-16 h-16 rounded-full overflow-hidden object-cover ${hidden ? 'hidden' : ''}"
+    src="./placeholders/family.jpg"
+    alt="A caring father, balancing one child on his shoulder and gently cradling another in his arms."
+  />
+`;
+
+const information = (hidden = false, text = '4xl') => (hidden ? '' : html`<p class="mt-4 text-sm">${text}</p>`);
+
+/**
+ * Default: This shows quote in its default state.
```

Comment:
The description should be changed to something describing more the use of patterns (IMO). From what I understand there is no "versions" or "states" in patterns right?

Reply:
like the idea...

Reply:
Sounds great! Some days ago we discussed the difference between samples and patterns and it feels like this could be the right moment to write that finally down.

I could take that over today!